sdgym/summary.py

```diff
@@ -173,7 +173,11 @@ def add_sheet(dfs, name, writer, cell_fmt, index_fmt, header_fmt):
 
         for idx, column in enumerate(df.columns):
             worksheet.write(startrow, idx, column, header_fmt)
-            width = max(len(column), *df[column].astype(str).str.len()) + 1
+            if df.empty:
```

Comment:
Should we add a unit test for this?

Reply:
Good point @amontanez24
At some point we may add unit tests to cover all this code. But in this case, since the rest of the code is not tested either, I think it is fine to go ahead and merge as it is.